lib/core/src/lib/dialogs/dialog/dialog.component.scss

```diff
@@ -168,6 +170,10 @@ $dialog-padding: 24px;
         bottom: 0;
     }
 
+    #{$mat-dialog-title}::before {
```

Comment:
I guess the long term goal is to get rid of mat-selectors completely along with the corresponding scss selector file. So instead of targeting mat-dialog-title, you can instead use the 'adf-dialog-header' class and align the text that way. Also, instead of using the ::before pseudo selector and setting its display property to none, a better approach would be to remove the justify-content property from the adf-dialog-header css, and give the adf-dialog-title a flex value of 1. So, all in all, the css would look something like 

```css
.adf-dialog-header {
        position: relative;
        display: flex;
        /* justify-content: space-between; <-Remove this */
        align-items: center;
        padding-bottom: $dialog-padding;

       ...

        .adf-dialog-title {
            font-size: large;
            font-weight: 200;
            margin: 0;
            flex: 1    /* <-Add this */
        }
        ...
}
```

Reply:
applied suggestion, thank you!